lib/opal/magic_comments.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 module Opal::MagicComments
-  MAGIC_COMMENT_RE = /\A# *(\w+) *: *(\w+) *$/.freeze
-  EMACS_MAGIC_COMMENT_RE = /\A# *-\*- *(\w+) *: *(\w+) *-\*- *$/.freeze
+  MAGIC_COMMENT_RE = /\A# *(\w+) *: *(\S+.*?) *$/.freeze
```

Comment:
ignore this comment if I'm overthinking, but the idea of using magic comments feels cryptic to me. Comments shouldn't be used to change behavior of your code. However Ruby does it, so I don't know if it's ok or not.

Also I think I'd personally expect these comments (if they are valuable at all) to be used as some kind of meta-information about the file (like by some external analyzer, rubocop is an example), and so I'd expect that I can safely remove them.

Is it that much worse writing `var <helper> = Opal.<helper>` at the beginning of the file?

Reply:
The way I see these helpers is that they're mostly for core stuff and very unlikely to be used elsewhere (although it wouldn't be a problem per se). Broadening the conversation to include also the `use_strict` comment I expect it to be transient and probably change it's default from `false` to `true` in `v1.2`. I personally dislike having to mark _every_ file with _frozen string  literal_ although I like the effect.

In summary `helpers` should only (mostly?) live within opal core, and `use_strict` is transient. Hope in this perspective it becomes less of a concern.